docs/rn/0.56.md

```diff
@@ -0,0 +1,46 @@
+# Release 0.56
+
+:material-calendar: 2024-07-09 · :material-list-status: [Full Changelog](https://github.com/srl-labs/containerlab/releases)
+
+## Interface aliases
+
+We know, it has been a long time coming, but thatnks to a brilliant contribution done by [@vista-](https://github.com/vista-) in #2124 we now have added support for [interface aliases](../manual/topo-def-file.md#interface-naming).
+
+What are interface aliases you ask? They are a way to use interface names as you see them in the NOS CLI, rather than the default `ethX` naming scheme. It is a convenience feature, but, oh boy, how much easier it is to define links when you don't have to keep doing this mental gymnastics of mapping `eth0` to `Gi0/1` and so on.
+
+--8<-- "docs/manual/topo-def-file.md:aliases"
+
+Interface aliases will be also displayed in the [generated graph](../cmd/graph.md#html) and even in the [edgeshark](../manual/wireshark.md#edgeshark-integration) web view.
+
+/// tab | Graph
+![pic1](https://gitlab.com/rdodin/pics/-/wikis/uploads/9ce9f8bd9ff7e4adfbe45c46712016b0/image.png){.img-shadow}
+///
+/// tab | Edgeshark
+![pic2](https://gitlab.com/rdodin/pics/-/wikis/uploads/3d4541b11a1b557967e419165aeb7ffd/image__2___1___1_.png){.img-shadow style=height:500px}
```

Comment:
oh wow, I didn't even realise this was a thing :)

Reply:
yeah, edgeshark is awesome